src/pages/lesson-details/LessonDetails.tsx

```diff
@@ -76,34 +78,45 @@ const LessonDetails = () => {
   const items = [
     {
       title: 'lesson.content',
-      content: <div dangerouslySetInnerHTML={{ __html: response.content }} />
+      content: (
+        <Box
+          dangerouslySetInnerHTML={{ __html: response.content }}
```

Comment:
Using `dangerouslySetInnerHTML` is dangerous in scope of XSS attacks for example. Maybe we need to use a package for it? For example `dompurify` is a good option used in many cases

Maybe would be useful: https://deadsimplechat.com/blog/how-to-safely-use-dangerouslysetinnerhtml-in-react/

Reply:
Thanks for the suggestion, I used the `DOMPurify` library and now it looks safer indeed.